.github/scripts/consolidate-snyk-prs.sh

```
@@ -0,0 +1,126 @@
+        echo "❌ Aborted by user"
+        exit 1
+    fi
+fi
+
+echo ""
+echo "🔒 Closing Snyk PRs..."
+echo ""
+
+CLOSED_COUNT=0
+
+echo "$SNYK_PRS" | while IFS=$'\t' read -r number title url; do
```

Shellcheck (0.11.0)

[warning] url appears unused. Verify use (or export if used externally).

(SC2034)